module/main.js

```diff
@@ -299,17 +299,17 @@ Hooks.once('init', async function () {
 
   Handlebars.registerHelper('getEdgeName', function (key) {
     const edges = {
-      arsenal: 'VTM5E.Arsenal',
-      ordnance: 'VTM5E.Ordnance',
-      library: 'VTM5E.Libraryy',
-      improvisedgear: 'VTM5E.ImprovisedGear',
-      globalaccess: 'VTM5E.GlobalAccess',
-      dronejockey: 'VTM5E.DroneJockey',
-      beastwhisperer: 'VTM5E.BeastWhisperer',
-      sensetheunnatural: 'VTM5E.SenseTheUnnatural',
-      repeltheunnatural: 'VTM5E.RepelTheUnnatural',
-      thwarttheunnatural: 'VTM5E.ThwartTheUnnatural',
-      artifact: 'VTM5E.Artifact'
+      arsenal: 'WOD5E.Arsenal',
+      ordnance: 'WOD5E.Ordnance',
+      library: 'WOD5E.Libraryy',
+      improvisedgear: 'WOD5E.ImprovisedGear',
+      globalaccess: 'WOD5E.GlobalAccess',
+      dronejockey: 'WOD5E.DroneJockey',
```

Comment:
This is the line where `super-linter` fails. I believe is a false positive, look at the log
```sh
2022-10-21 22:53:04 [INFO]   ---------------------------
2022-10-21 22:53:04 [INFO]   File:[/github/workspace/module/main.js]
2022-10-21 22:53:04 [ERROR]   Found errors in [gitleaks] linter!
2022-10-21 22:53:04 [ERROR]   Error code: 1. Command output:
------

    ○
    │╲
    │ ○
    ○ ░
    ░    gitleaks

Finding:     dronejockey: 'WOD5E.DroneJockey',
Secret:      WOD5E.DroneJockey
RuleID:      generic-api-key
Entropy:     3.734522
File:        /github/workspace/module/main.js
Line:        307
Fingerprint: /github/workspace/module/main.js:generic-api-key:307

10:53PM INF scan completed in 22.1ms
10:53PM WRN leaks found: 1
------
```

It means nothing of usefullness, am I right? Superlinter shouldn't be an issue here. Can we manually lint the linters we want?

Reply:
I finally could run locally `gitleaks` on my machine.
It seems the problem is gitleaks see the word `key` in `jockey` and freaks out
![image](https://user-images.githubusercontent.com/30557287/206772258-0e551002-8634-4989-98b0-66e53a7054b6.png)

Reply:
> Yea. That would be a great addition in the PR. Having a lint ignore comment here would work as well.

I tried to [ignore it with the comment](https://github.com/zricethezav/gitleaks#gitleaksallow) `//gitleaks:allow` but I have the same result. I believe we have to drop `super-linter` and use normal `eslint`. Is that allright to you?

Reply:
I chose super-linter as a one size fits all solution to linting which has its ups and downs. Since we aren't doing anything key related with a FoundryVTT system, we can add .gitleaksignore to the ignore file.
A better solution (if you're up to it) is to change the workflows to use eslint as you mentioned but to include HTML linting, CSS linting, etc.

Reply:
Allright, I'll change the lints in my current PR. I will keep the same lint engines chosen by super-linter, so we shouldn't have any noticeable difference 👍🏼